docs/README.md

```diff
@@ -46,24 +46,8 @@ The following JSON is the default configuration for the module and can be overri
 }
 ```
 
-## Using Yoast SEO as an alternative
+## Using Yoast SEO Premium
 
-Depending on your needs you may wish to install [Yoast SEO](https://yoast.com/wordpress/plugins/seo/) for its advanced features. It is recommended to disable the Metadata and XML Sitemaps components if you do this using the following configuration.
-
-```json
-{
-	"extra": {
-		"altis": {
-			"modules": {
-				"seo": {
-					"enabled": true,
-					"metadata": false,
-					"xml-sitemaps": false,
-				}
-			}
-		}
-	}
-}
-```
+[Yoast SEO Premium](https://yoast.com/wordpress/plugins/seo/) adds more features and access to Yoast's support team. Altis SEO is configured in such a way that if you own a copy of Yoast SEO Premium, all you need to do is install it as a plugin or mu-plugin normally and it will work seamlessly without any additional configuration.
```

Comment:
Looking at Yoast's own documentation for installing using composer here and it's not very helpful, or not as helpful as it could be, it'd be good to provide advice on using composer as the recommended approach.

https://yoast.com/help/how-to-install-yoast-plugins-using-composer/

```suggestion
[Yoast SEO Premium](https://yoast.com/wordpress/plugins/seo/) adds more features and access to Yoast's support team. Altis SEO is configured in such a way that if you own a copy of Yoast SEO Premium, all you need to do is install it as a plugin or mu-plugin normally and it will work seamlessly without any additional configuration.

We recommend installing Yoast SEO Premium using Composer. To do this follow the instructions linked to in the [Downloads section of MyYoast](https://my.yoast.com/downloads). You will be able to create a developer token and then be provided with the required `composer.json` updates and commands to run.
```